web/forms.py

```
@@ -1842,3 +1844,21 @@ class StudyGroupForm(forms.ModelForm):
+        widget=TailwindInput(attrs={"placeholder": "Enter survey title"}),
+        help_text="Give your survey a clear and descriptive title",
+    )
+
+    class Meta:
+        model = Survey
+        fields = ["title"]
+
+    def clean_title(self) -> str:
+        title = self.cleaned_data.get("title")
+        if len(title) < 5:
+            raise forms.ValidationError("Title too short", code="invalid_length", params={"min_length": 5})
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)